gen/names.go

```diff
@@ -86,8 +86,7 @@ func (g *nameGen) generate() (string, error) {
 				name = "R" + name
 			}
 			if !token.IsIdentifier(name) {
-				return "", errors.Wrapf(&ErrNotImplemented{Name: "crypticName"},
-					"can't generate valid name: %+v", g.parts)
+				return "", errors.Errorf("can't generate valid name: %+v", g.parts)
```

Comment:
The error wrapping here was causing the output to say that the "crypticName" feature was not implemented. The actual error and stack were not being output.